web/react-web/src/Graph/Graph.js

```diff
@@ -0,0 +1,241 @@
+import React, {Component} from 'react';
+import Highcharts from 'highcharts'
+import HighchartsReact from 'highcharts-react-official'
+import {AXES_LINEAR, AXES_LOG, AXES_LOG_LOG} from "../sharedObjects";
+
+const colors = ["#7cb5ec", "#434348", "#90ed7d", "#f7a35c", "#8085e9", "#f15c80", "#e4d354", "#2b908f", "#f45b5b", "#91e8e1"];
+
+const predefinedOptions = {
+    chart: {
+        zoomType: 'x',
+        colors,
+    },
+    title: {
+        text: ''
+    },
+    subtitle: {
+        text: document.ontouchstart === undefined ?
+            'Click and drag in the plot area to zoom in' : 'Pinch the chart to zoom in'
+    },
+    tooltip: {
+        shared: true,
+        useHTML: true
+    },
+    xAxis: {
+        type: 'linear',
+        labels: {},
+        title: {},
+        crosshair: {
+            width: 10,
+            color: 'rgba(0, 0, 0, 0.1)'
+        }
+    },
+    yAxis: {
+        type: 'linear',
+        title: {
+            text: 'Active cases'
+        }
+    },
+    legend: {
+        enabled: true,
+        useHTML: true,
+        itemWidth: 300
+    },
+    plotOptions: {
+        series: {
+            animation: false
+        },
+        line: {
+            marker: {
+                symbol: 'circle'
+            }
+        }
+    },
+
+    series: []
+};
+
+class Graph extends Component {
+
+    static defaultProps = {
+        options: {},
+        data: []
+    };
+
+    render() {
+        const {
+            data,
+            options
+        } = this.props;
+
+        const {
+            axesType,
+            xAxisRelative,
```

Comment:
Minor: We can consider `isXAxisRelative` to make it clearer it is a bool.

Reply:
Changed.